### PR TITLE
Standardize `ns` formatting in all namespaces

### PR DESCRIPTION
**Pull request by noahmoss**
This is a mega-PR to reformat `ns` forms in almost all Clojure namespaces in the repository according tot the [How to ns](https://stuartsierra.com/2016/clojure-how-to-ns.html#line-breaks) guide. Some relevant discussion is in [this thread](https://metaboat.slack.com/archives/CKZEMT1MJ/p1670955776517399).

To do this I (programmatically) opened each file in Vim and ran `:IcedCleanNs`, a [command in the vim-iced plugin](https://liquidz.github.io/vim-iced/vim-iced.html#%3AIcedCleanNs) which uses refactor-nrepl's [clean-ns operation](https://github.com/clojure-emacs/refactor-nrepl#clean-ns) under the hood.

The major changes are
* A newline always after the `:require` and `:import` keywords
* Using lists inside of `:import` instead of vectors
* Putting every `:require`d namespace in a vector, and all `:import`ed packages in a list
* Some long lines split into multiple lines (usually by putting `:refer` on a new line)

There are a couple of small unrelated indentation/formatting changes in the as a result of my editor opening the files but it should all be a no-op.

I've scrolled through the diff to check that everything is sane, but if anyone else wants to do that too, [this JS snippet](https://gist.github.com/juanca/5fd799c5b094e3e4f8b709cd101d7403) is handy for forcing GitHub to load everything.

### CODE REVIEW

File: bin/i18n/src/i18n/common.clj [hunk 1]
```diff
@@ -1,7 +1,8 @@
 (ns i18n.common
-  (:require [clojure.java.io :as io]
-            [clojure.string :as str]
-            [metabuild-common.core :as u])
+  (:require
+   [clojure.java.io :as io]
+   [clojure.string :as str]
+   [metabuild-common.core :as u])
   (:import [org.fedorahosted.tennera.jgettext Catalog HeaderFields Message PoParser]))
```
Comment (qnkhuat) [inline bin/i18n/src/i18n/common.clj:6]: nit: it'd be nice to have these `import` in a new line too?

**Review by qnkhuat — APPROVED** [review]
1. I did my best reviewing this!
Screenshot 2022-12-29 at 15 07 12
2. there are some imports that were still wrong, I pushed a commit to fix that.
3. I wonder if clj-kondo has rules to enforce this
4. did you use vim macro?

**Comment by noahmoss** [comment]
@qnkhuat thanks for the review and the fixes!

Sort of...didn't use a macro exactly but made a couple keybindings to connect to the repl, run `:IcedCleanNs` and write a file. Then opened all clojure files in the arglist and iterated through them, running the command manually on each one. So not quite fully automated but I wanted a bit of manual oversight to make sure everything was working.

Not sure about kondo, good question

File: enterprise/backend/src/metabase_enterprise/enhancements/integrations/ldap.clj [hunk 2]
```diff
@@ -46,9 +50,9 @@
                                 (when-not (= syncable-attributes (:login_attributes user))
                                           {:login_attributes syncable-attributes})
                                 (when (not= first-name old-first-name)
-                                          {:first_name first-name})
+                                      {:first_name first-name})
```
Comment (camsaul) [inline enterprise/backend/src/metabase_enterprise/enhancements/integrations/ldap.clj:53]: This indentation looks busted

File: enterprise/backend/src/metabase_enterprise/enhancements/integrations/ldap.clj [hunk 3]
```diff
@@ -46,9 +50,9 @@
...
                                 (when (not= last-name old-last-name)
-                                          {:last_name last-name}))]
+                                      {:last_name last-name}))]
```
Comment (camsaul) [inline enterprise/backend/src/metabase_enterprise/enhancements/integrations/ldap.clj:55]:
```suggestion
                                  {:last_name last-name}))]
```

File: enterprise/backend/src/metabase_enterprise/enhancements/integrations/ldap.clj (hunk 2 above)
Comment (noahmoss) [reply enterprise/backend/src/metabase_enterprise/enhancements/integrations/ldap.clj:53]: oh you're right. the indentation of the whole function was already messed up actually, I'll put up a quick PR to fix that

Comment (noahmoss) [reply enterprise/backend/src/metabase_enterprise/enhancements/integrations/ldap.clj:53]: https://github.com/metabase/metabase/pull/27467

**Pull request merged**